### PR TITLE
Fixed About and added github button

### PR DESCRIPTION
**Pull request by Garg-Doppler**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -280,16 +270,37 @@ <h4 style="font-family: 'Exo', sans-serif; text-align: center;">
...
+                    <div class="card-body">
+                        <h4 style="font-family: 'Exo', sans-serif; text-align: center;">
+                            Decision Trees
+                        </h4>
+                        <p style="font-family: 'Exo', sans-serif; text-align: center;">
+                            Visualise the separation of classes using Decision Trees.
+                        </p>
+                    </div>
+                </div>
+            </div> -->
```
Comment (AnirudhDagar) [inline index.html:303]: Can we remove the commented code?

Comment (Garg-Doppler) [reply index.html:303]: Sure, but it is just to make tasks easier in the future. We can just uncomment this piece to make the box for decision trees whenever it is complete. However, I will remove it if you wish.

Comment (AnirudhDagar) [reply index.html]: We can add that back later.

**Comment by AnirudhDagar** [comment]
Thanks!

**Pull request merged**